<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editprofileemp</class>
 <widget class="QDialog" name="editprofileemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_updateemp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1261</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_cemp">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_empcn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_empdes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_emppn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_empad">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_desc">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phno">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveemp">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_suces">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_update">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_jhlogo_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_jhlogo_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_update">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eppf">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
